purchases/src/main/kotlin/com/revenuecat/purchases/InternalRevenueCatAPI.kt

```diff
@@ -0,0 +1,15 @@
+package com.revenuecat.purchases
+
+@Retention(value = AnnotationRetention.BINARY)
+@RequiresOptIn(
+    level = RequiresOptIn.Level.ERROR,
+    message = "This is an internal RevenueCat API that may change frequently and without warning. " +
+        "No compatibility guarantees are provided. It is strongly discouraged to use this API.",
+)
+@Target(
+    AnnotationTarget.CLASS,
+    AnnotationTarget.FUNCTION,
+    AnnotationTarget.PROPERTY,
+    AnnotationTarget.TYPEALIAS,
+)
+annotation class InternalRevenueCatAPI
```

Comment:
Following the [Kotlin](https://kotlinlang.org/docs/coding-conventions.html#choose-good-names) and [Android](https://developer.android.com/kotlin/style-guide#camel_case) naming conventions, this would be `InternalRevenueCatApi`, but I made "API" uppercase for consistency with our existing `ExperimentalPreviewRevenueCatPurchasesAPI`. Let me know what you think.